src/bundle/Resources/public/scss/_user-menu.scss

```diff
@@ -58,15 +81,22 @@
         &__item {
             width: 100%;
             cursor: pointer;
-            padding: 0 1.5rem;
+            padding: 0 calculateRem(16px);
 
             .nav-link {
+                display: grid;
+                grid-template-columns: calculateRem(20px) 1fr;
+                grid-column-gap: calculateRem(8px);
+                grid-template-areas: 'icon label';
+                padding: calculateRem(4px) calculateRem(4px);
+
                 color: $ez-black;
                 font-size: calculateRem(14px);
-            }
 
-            &:not(:last-child) {
-                border-bottom: 1px solid $ez-ground-base-dark;
+                .ez-icon--small-medium {
```

Comment:
Duplicated values, to check.

Reply:
It overwrites this: https://github.com/ezsystems/ezplatform-admin-ui/commit/a323f42fafb589460e636f3ad51e76baaa875773#diff-4a830143277c2bbd9e1d3228c2c048c1R27
But I guess we can delete it?

Reply:
I think it was before we started to use the modifiers. You can, but you also have to spot all the places where the `--medium` class should be applied.